src/components/button/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./button";
```

Comment:
this file is not required as it can be easily handled in `components/index.ts`

Reply:
- This is intentionally setup like this, you can read more about it in #4